### ISSUE TITLE
History Support for Bani Controller

### ISSUE THREAD

**Issue by maneetpaul** [KhalisFoundation/sttm-web#1057]
Bani Controller is no longer displaying history like it used to when we launched. 

This history should stay in Sync with the history on desktop and vice versa

**Comment by saintsoldierx** [comment]
I still don't think history has been added to Bani Controller? @Gauravjeetsingh @maneetpaul ?

**Comment by gurjit03** [comment]
@Gauravjeetsingh please can you update on this one or close if done.

**Comment by gurjit03** [comment]
@Gauravjeetsingh veerji, any update on this ? Should we keep it open or close it?

**Comment by Gauravjeetsingh** [comment]
Yeah, need some clarity on this one. Is this about adding history of shabads or history of which verses are read in a shabad?

If its about verses, that should be working, and we can work out few edge cases, where it fails. For history of shabads, I dont remember we ever had that.

**Comment by gurjit03** [comment]
@Gauravjeetsingh sure, can we try to get this needed information, so we can proceed further accordingly.

**Comment by saintsoldierx** [comment]
@gurjit03 I would say both use cases are valid
